backend/app/controllers/loginController.js

```diff
@@ -0,0 +1,97 @@
+const db = require("../models");
+
+exports.loginWithGoogle = (req, res) => {
+    // Validate request
+    if (!req.body.id) {
+        res.status(400).send({
+            message: "Content can not be empty!"
+        });
+        return;
+    }
+    db.google_account.findByPk(req.body.id, {
+        include: [
```

Comment:
what is the reason of this include?

Reply:
Are you using that user during the account creation?

Reply:
the result of this method will be used after login successful. User information will be used in home page